operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/InformerConfiguration.java

```diff
@@ -0,0 +1,62 @@
+package io.javaoperatorsdk.operator.processing.event.source.informer;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.javaoperatorsdk.operator.api.config.ConfigurationService;
+import io.javaoperatorsdk.operator.api.config.DefaultResourceConfiguration;
+import io.javaoperatorsdk.operator.processing.event.ResourceID;
+import io.javaoperatorsdk.operator.processing.event.source.AssociatedSecondaryResourceIdentifier;
+import io.javaoperatorsdk.operator.processing.event.source.PrimaryResourcesRetriever;
+
+public class InformerConfiguration<R extends HasMetadata, P extends HasMetadata>
+    extends DefaultResourceConfiguration<R> {
+
+  private final PrimaryResourcesRetriever<R> secondaryToPrimaryResourcesIdSet;
+  private final AssociatedSecondaryResourceIdentifier<P> associatedWith;
+  private final boolean skipUpdateEventPropagationIfNoChange;
+
+  public InformerConfiguration(ConfigurationService service, String labelSelector,
```

Comment:
The configuration service is a little bit weird here a little bit. Especially if somebody wants to create an Informer without Depdendent resources.